tests/Jenkins/dirac-cfg-update.py

```diff
@@ -34,7 +40,16 @@
 if cFile:
   localConfigFile = cFile
 else:
-  localConfigFile = './etc/dirac.cfg'
+  print "WORKSPACE: %s" % os.path.expandvars('$WORKSPACE')
+  if os.path.isfile( os.path.expandvars('$WORKSPACE')+'/PilotInstallDIR/etc/dirac.cfg' ):
+    localConfigFile = os.path.expandvars('$WORKSPACE')+'/PilotInstallDIR/etc/dirac.cfg'
+  elif os.path.isfile( os.path.expandvars('$WORKSPACE')+'/ServerInstallDIR/etc/dirac.cfg' ):
+    localConfigFile = os.path.expandvars('$WORKSPACE')+'/ServerInstallDIR/etc/dirac.cfg'
+  elif os.path.isfile( './etc/dirac.cfg' ):
+    localConfigFile = './etc/dirac.cfg'
+  else:
```

Comment:
"multiple identities" Raja... suspicious! ;-)

Anyway, on the comment: this is for test purposes, no need for further extension, IMHO an utility here would not make things simpler I am afraid.